cmd/contour/serve.go

```diff
@@ -303,9 +316,21 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	eh.Metrics = metrics
 	eh.CacheHandler.Metrics = metrics
 
+	synced := make([]cache.InformerSynced, 0, len(informers))
+	for _, inf := range informers {
```

Comment:
Possibly move these four lines closer to where they are used inside the step 13 function below.